packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -69,25 +69,25 @@ export class Breadcrumb extends BaseComponent<IBreadcrumbProps, IBreadcrumbState
     let overflowMenuId = this._id + '-overflow';
 
     return (
-      <div className={ css('ms-Breadcrumb', className) } ref='renderingArea'>
+      <div className={ css('ms-Breadcrumb', className, styles.root) } ref='renderingArea'>
```

Comment:
The concern is that when we remove classnames, we break implicit contracts. We don't want to do that just yet, until we have a viable alternative for customization.